src/frontend-samples/issues-sample/IssuesWidget.tsx

```diff
@@ -454,19 +484,39 @@ const IssuesWidget: React.FunctionComponent = () => {
     ));
   };
 
+  const filterTypes = ["All", "Civil Design", "Clash", "Closeout", "Communication", "Deficiency", "Design", "Field Data", "Issue", "Observation", "Other", "Punchlist", "Risk", "RFI", "Submittal Issue", "Task", "Transmittal Issue"];
```

Comment:
For several of these filter types, there are no matching issues in the sample iModel.  Why do we need so many in the picker?  It might make sense if they list was being retrieved from some API, but if we are hardcoding the list, I suggest just leaving out most or all of the ones with no issues.

Reply:
Only the filters that return a result are displayed now.